pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -720,6 +732,53 @@ func (c *Cluster) getPVCMetadataInitContainerActivate(mountPath string, osdProps
 	}
 }
 
+func (c *Cluster) getPVCWaldataInitContainer(mountPath string, osdProps osdProperties) v1.Container {
```

Comment:
@leseb Since we are using raw mode for the PVs, do we still need the init container to copy it? I thought we didn't need it anymore.

Reply:
Do you means the funcions: getPVCInitContainer, getPVCMetadaInitContainer() and getPVCWalInitContainer() are not needed to append in InitContainers()?

Reply:
after rebase, I followed the process with metadata device for PVC. can we put the optimization (remove the initcontainers, it seems some changes) after this PR merged?

Reply:
rebased again, it seems some re-construction for the initContainers, and also update wal device part.

Reply:
set it resolved after rebased, waiting for new comments.